<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DYELogAnalyzeToolDlg</class>
 <widget class="QWidget" name="DYELogAnalyzeToolDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DYELogAnalyzeToolDlg</string>
  </property>
  <widget class="QPushButton" name="PUSHBUTTON_OPENFILEDLG">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PUSHBUTTON_ANALYZETIMECONSUME">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>分析耗时操作</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PUSHBUTTON_ANALYZEFAILSERVICE">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>分析业务失败</string>
   </property>
  </widget>
  <widget class="QLabel" name="LABEL_FILEFULLPATH">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>文件路径：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LINEEDIT_FILEFULLPATH">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>521</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="LISTWIDGET_SHOWANALYZERESULT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>861</width>
     <height>261</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QListWidget" name="LISTWIDGET_SHOWTXTCONTEXT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>861</width>
     <height>301</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
